packages/lesswrong/themes/eaForumTheme.js

```diff
@@ -4,11 +4,17 @@ import deepOrange from '@material-ui/core/colors/deepOrange';
 
 const sansSerifStack = [
   'Raleway',
+  'Helvetica Neue',
```

Comment:
On a regular post this kinda clashes with the text. Why is the title san serif anyway?